js/flightlog_fields_presenter.js

```diff
@@ -750,8 +750,11 @@ function FlightLogFieldPresenter() {
                 return (value / 1024 * 100).toFixed(2) + "%";
 
             case 'headspeed':
+                Rotorspeed = (value).toFixed(0); 
```

Comment:
This is probably not the right place to catch the rotorspeed. This function is a general formatting function, and we don't know if it is called in a way that sets the rotorspeed to a correct value.